cpskin/workflow/upgradehandlers.py

```diff
@@ -5,6 +5,10 @@
 from zope.ramcache.interfaces.ram import IRAMCache
 
 
+def add_private_view(context):
+    context.runAllImportStepsFromProfile('profile-cpskin.workflow:default')
```

Comment:
Not too dangerous to reinstall everything ?
It scares me sometimes 😄

Reply:
It's just workflow package, usually nobody change portal workflow ;-)